interface/themes/style_light.css

```diff
@@ -759,10 +759,11 @@ ul.tabNav li {
 }
 ul.tabNav li.current a {
   background: #ffffff;
+  font-weight: bold;
   color: #2672ec;
 }
 ul.tabNav a {
-  color: #ffffff;
+  color: #2672ec;
```

Comment:
@jonathanzhang53 this works cause the themes bg is white. This will not be true in all cases... if the user edits this in user preferences or global. Nevertheless goo. Tested an it works